libethereum/SkaleHost.cpp

```diff
@@ -592,12 +588,14 @@ void SkaleHost::createBlock( const ConsensusExtFace::transactions_vector& _appro
                 EIP1559TransactionsPatch::isEnabledInWorkingBlock(),
                 InvalidTransactionFormatPatch::isEnabledInWorkingBlock() );
 #ifdef BITE
-            auto it = _decryptedTransactionDataFields->find( i );
-            if ( it != _decryptedTransactionDataFields->end() ) {
-                t.setDecryptedData( it->second );
-                if ( it->second )
-                    validDecryptedTransactionDataFields->insert( *it );
+            auto it = _decryptedTransactionFields->find( i );
+            if ( it != _decryptedTransactionFields->end() ) {
+                DecryptedTransactionFields& txFields = it->second;
+
```

Comment:
we should allow non-decryptable transactions to pass consensus stage and arrive to skaled. skaled then charges users who submitted those transactions for submitting invalid transactions on purpose